### ISSUE TITLE
Reporting of pre-run FS in fs-subjects-dir in reports

### ISSUE THREAD

**Issue by utooley** [poldracklab/fmriprep#2049]
Version: 20.0.2
See Neurostars [post](https://neurostars.org/t/detection-of-fs-subjects-dir-in-20-0-2/6342) about this.

I have pre-run Freesurfer and pointed fMRIprep to the directory using `--fs-subjects-dir`, but the report still says “FreeSurfer reconstruction: Run by fMRIPrep”.  I'm almost positive that fMRIPrep is detecting the pre-run surfaces, given the contents of the working dir and the log, but the reporting is wrong, perhaps due to the lines [here](https://github.com/poldracklab/fmriprep/blob/0d0f62e0132d648867a8ecbb72c7670a9e77cf33/fmriprep/interfaces/reports.py#L102-L113)? I'm not sure I understand what's going on there with deciding based on `echo`, though.

`_inputs.pklz` in `working_dir/fsdir_run_20200323_143435_c23f0b8d_4707_4593_9b85_dc891f5fefe5` has the correct Freesurfer subjects directory:

```
Singularity fmriprep-20-0-2.simg:/mnt> nipypecli show _inputs.pklz 
{'derivatives': '/mnt/derivatives/fmriprep_t2',
 'freesurfer_home': '/opt/freesurfer',
 'overwrite_fsaverage': False,
 'spaces': [],
 'subjects_dir': '/mnt/derivatives/freesurfer_t2'}
```

And the logs show something like this (too fast for Freesurfer to run!)
```
[Node] Finished "fmriprep_wf.single_subject_CBPD0000_wf.func_preproc_ses_02_task_rest_run_02_wf.bold_bold_trans_wf.bold_reference_wf.enhance_and_skullstrip_bold_wf.pre_mask_dilate".
200323-16:31:13,460 nipype.workflow INFO:
	 [Node] Setting-up "fmriprep_wf.single_subject_CBPD0000_wf.anat_preproc_wf.surface_recon_wf.autorecon1" in "/tmp/fmriprep_wd/fmriprep_wf/single_subject_CBPD0000_wf/anat_preproc_wf/surface_recon_wf/autorecon1".
200323-16:31:13,651 nipype.interface INFO:
	 recon-all complete : Not running
200323-16:31:13,652 nipype.workflow INFO:
	 [Node] Running "autorecon1" ("smriprep.interfaces.freesurfer.ReconAll"), a CommandLine Interface with command:
echo recon-all: nothing to do
200323-16:31:13,654 nipype.interface INFO:
	 recon-all complete : Not running
200323-16:31:14,626 nipype.workflow INFO:
	 [Node] Finished "fmriprep_wf.single_subject_CBPD0000_wf.anat_preproc_wf.surface_recon_wf.autorecon1".
```

**Comment by ariekahn** [comment]
It's deciding on the message by the freesurfer nipype interface doing a quick check for whether all its outputs already exist, and if so, the command should return `"echo recon-all: nothing to do"`.

Was the existing directory you're using generated by fmriprep in the past, or by directly running recon-all? It looks like `smriprep` skips a few processing steps, so when the recon-all interface checks for outputs, it's missing a few output steps, specifically `careginv`, `rmneck`, `skull-lta`, `cortparc3`, and `balabels`. 

So basically, the check is out of sync with the smriprep processing. The simplest thing might just be to only check for outputs we care about, by using `flags='-noskullstrip -nobalabels -nocareginv -normneck -noskull-lta -nocortparc3'`.

**Comment by utooley** [comment]
I did investigate this a bit more afterwards--it was generated by directly running recon-all, but this check has worked fine on those kinds of Freesurfer directories in the past. And the logs do have 
```
200323-16:31:13,652 nipype.workflow INFO:
[Node] Running "autorecon1" ("smriprep.interfaces.freesurfer.ReconAll"), a CommandLine Interface with command:
echo recon-all: nothing to do
```
in them. The check for existing outputs (so as not to rerun FS) is working, Freesurfer doesn't run again, just the report output is wrong. Perhaps because the command no longer starts with `echo`, for some reason, after all is said and done?
